### DISCUSSION TITLE
Pre-mask inpainting images

### DISCUSSION THREAD

**Discussion by hammeiam** [lllyasviel/Fooocus#1222] [Ideas]
Is it possible to mask an image in Photoshop and bring it directly into Fooocus without needing to use the tool's inpainting UI? It works very well, but other tools are specialized for selecting or negating selections.

**Comment by hammeiam** [comment]
Additionally I'd love to be able to download masked images for reuse if the above isn't possible.

**Comment by xhoxye** [comment]
![QQ截图20231206210617](https://github.com/lllyasviel/Fooocus/assets/129571231/2e414499-4159-4baf-aafc-84bb2af26b21)

And you might want to try this.
https://github.com/lllyasviel/Fooocus/pull/1126
